docs/brew-tap.md

```diff
@@ -73,7 +73,7 @@ If you need a formula to be installed from a particular tap, you can use fully
 qualified names to refer to them.
 
 For example, you can create a tap for an alternative `vim` formula. Without
-pinning it, the behavior will be
```

Comment:
This one was correct. It’s like color vs colour — the former is American English, and the latter is used in the rest of the world. Don’t the docs primarily use American English spelling?